assets/svgReactNative/missing_details/Ve1.js

```diff
@@ -1,5 +1,5 @@
-import * as React from 'react';
-import Svg, { G, Path } from 'react-native-svg';
+import * as React from "react";
```

Comment:
This happens all the time, can we decide on single or double quote? My linter says single all the time

Reply:
Should be single quote according to the linter setting for this project